servant/src/Servant/API/Methods.hs

```diff
@@ -0,0 +1,12 @@
+{-# LANGUAGE DataKinds          #-}
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE KindSignatures     #-}
+{-# OPTIONS_HADDOCK not-home    #-}
+
+module Servant.API.Methods where
+
+import           Data.Typeable (Typeable)
+import           GHC.TypeLits (Nat, Symbol)
+
```

Comment:
Maybe some documentation saying you can use this to define your own status code response/method?